### PR TITLE
Load users

### LINKED ISSUE #2: as an admin i want to see a list of all users from database and edit them by id
- [ ] load user details from DB to crm  
- [ ] edit user details by id

### PR DESCRIPTION
**Pull request by MohamadD**
load all users from DB and edit user by id closes #2 #3 #4
Added tow queries to the BE to handle the requests .

### CODE REVIEW

**Comment by Karyum** [comment]
@MohamadD 
- add description to the PR of what you did and the issue
- also merge with the latest master so i see fewer changed files

File: package.json [hunk 1]
```diff
@@ -30,5 +33,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "react-admin": "^3.0.4"
```
Comment (Karyum) [inline package.json]: react admin is a dependency not a dev dependency, so uninstall it and install as a dependency

File: src/App.jsx [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+import simpleRestProvider from "ra-data-simple-rest";
+import { UserList, UserEdit } from "./users";
 
-function App() {
+const App = () => {
   return (
-    <div>
-      <h1>Helo</h1>
-    </div>
+    <Admin dataProvider={simpleRestProvider("http://localhost:8000/admin")}>
```
Comment (Karyum) [inline src/App.jsx]: change this localhost to an environment variable of our api, ask your team if you need help with this
## DO NOT PUSH THE CONFIG.ENV FILE

File: src/users.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  EditButton,
+  Edit,
+  SimpleForm,
+  TextInput,
+  useMutation,
+  Button
+} from "react-admin";
+
+const ApproveButton = ({ record }) => {
+  console.log(record);
```
Comment (Karyum) [inline src/users.js]: let's remove this for now

File: src/users.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  const [approve, { loading }] = useMutation({
+    type: "update",
+    resource: "comments",
+    payload: { id: record.id, data: { isApproved: true } }
+  });
+  return <Button label="Approve" onClick={approve} disabled={loading} />;
```
Comment (Karyum) [inline src/users.js]: in here we need to check if it's already approved then show a text saying Approved

File: src/users.js (hunk 3 above)
Comment (MohamadD) [reply src/users.js]: fixed

File: src/users.js (hunk 4 above)
Comment (MohamadD) [reply src/users.js]: i totally changed that in the new branch

**Pull request merged**